apps/demo/src/app/drivers/magento/magento.module.ts

```diff
@@ -39,12 +45,13 @@ export class DemoMagentoDriverModule {
               console.log(
                 `[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}`,
               ),
-            );
+						);
           if (networkError) console.log(`[Network error]: ${networkError}`);
-        }),
+				}),
+				this.apolloCacheableOperationsLinkGenerator.getLink(),
```

Comment:
variable name should probably be -- magtentoLinkGen.getLink()

Reply:
changed